stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -811,6 +811,13 @@ for func in (:log, :sqrt)
     end
 end
 
+# Cube root of a real-valued symmetric matrix
+function cbrt(A::Symmetric{T}) where {T<:Real}
```

Comment:
We should also support `Hermitian{<:Real}` here.  (It's an unfortunate historical artifact that Julia has both `Hermitan` and `Symmetric` types, since they are redundant for real types and you can't do much with complex-symmetric matrices).  There is a `HermOrSym` alias in LinearAlgebra to help with this:
```suggestion
function cbrt(A::HermOrSym{<:Real})
```